Libplanet/Action/IContext.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+namespace Libplanet.Action
+{
+    public class IContext
```

Comment:
`IContext` seems a little confusing name. Is `IExecutionContext` too messy?

Reply:
I suggest `IActionContext` for an alternative name.

Reply:
It should be `interface` instead of `class`.